base/src/main/java/org/mozilla/jss/netscape/security/x509/OIDMap.java

```diff
@@ -132,6 +137,8 @@ private static void loadNamesDefault(Properties props) {
         props.put(AUTH_KEY_IDENTIFIER, "2.5.29.35");
         props.put(SUBJ_DIR_ATTR, "2.5.29.9");
         props.put(EXT_KEY_USAGE, "2.5.29.37");
+        props.put(EXT_INHIBIT_ANY_POLICY, "2.5.29.54");
+        props.put(OCSP_NO_CHECK, "1.3.6.1.5.5.7.48.1.5");
```

Comment:
This is also already defined in `OCSPNoCheckExtension.OID`. Should we use that one?